app/src/main/java/org/mozilla/focus/settings/ExperimentsSettingsFragment.kt

```diff
@@ -9,23 +9,37 @@ import android.os.Bundle
 import android.support.v14.preference.SwitchPreference
 import android.support.v7.preference.PreferenceFragmentCompat
 import com.jakewharton.processphoenix.ProcessPhoenix
+import mozilla.components.service.fretboard.ExperimentDescriptor
+import mozilla.components.service.fretboard.Fretboard
+import org.mozilla.focus.FocusApplication
 import org.mozilla.focus.R
 import org.mozilla.focus.web.Config
 import org.mozilla.focus.web.ENGINE_PREF_STRING_KEY
 
+const val EXPERIMENTS_JSON_FILENAME = "experiments.json"
+const val EXPERIMENTS_BASE_URL = "https://settings.prod.mozaws.net/v1"
+const val EXPERIMENTS_BUCKET_NAME = "main"
+const val EXPERIMENTS_COLLECTION_NAME = "focus-experiments"
+
+val experimentDescriptor = ExperimentDescriptor(Config.EXPERIMENT_DESCRIPTOR_GECKOVIEW_ENGINE)
```

Comment:
nit: Over time we will probably create a bunch of descriptors that we use in various places of the app. Should we centralized them somewhere? Maybe in an `Experiments` object? `Experiments.GECKOVIEW` ?